510_Deployment/40_config.asciidoc

```diff
@@ -3,7 +3,7 @@
 Elasticsearch ships with _very good_ defaults,((("deployment", "configuration changes, important")))((("configuration changes, important"))) especially when it comes to performance-
 related settings and options.  When in doubt, just leave
 the settings alone.  We have witnessed countless dozens of clusters ruined
-by errant settings because the administrator thought he could turn a knob
+by errant settings because the administrator thought they could turn a knob
```

Comment:
Perhaps he/she would be better. "they" specifies plural, so it will go with "administrators" rather. Current state is okay IMO.

Reply:
'they' as a gener-neutral singular pronoun is grammatically correct https://en.wikipedia.org/wiki/Singular_they

Reply:
Okay, great then. Wasn't aware of such a usage. (y)